cellprofiler/modules/measureobjectintensity.py

```diff
@@ -392,8 +403,24 @@ def run(self, workspace):
                         cmi_y[lindexes - 1] = i_y / integrated_intensity[lindexes - 1]
                         diff_x = cm_x - cmi_x[lindexes - 1]
                         diff_y = cm_y - cmi_y[lindexes - 1]
-                        mass_displacement[lindexes - 1] = \
-                            np.sqrt(diff_x * diff_x + diff_y * diff_y)
+                        if dimensions is 2:
```

Comment:
We should consider adding a property to `cellprofiler.image.Image` to clarify this common test.

We could add a `volumetric` property that returns a Boolean.